main/REPL/spawner.md

```diff
@@ -0,0 +1 @@
+**tyg todo: Need pointer to where this and commands are defined/explained**
```

Comment:
This is defined here: https://github.com/Agoric/agoric-sdk/blob/master/packages/spawner/src/contractHost.js

The user doesn't really use this from the REPL. It's used in the deploy scripts to deploy a long-lived backend for dapps (often defined in handler.js with deploy script in the api/ folder of dapps). This was the original contractHost before we had Zoe, so there are many features that it has that are not useful for its current purpose. It needs to be rewritten, so we shouldn't draw attention to it if possible.

Reply:
Moved down to "you shouldn't use this, so we're not telling you how to" section.